tests/ecdhp256-test.c

```diff
@@ -31,6 +35,13 @@ int main()
 {
 	uint8_t* result = (uint8_t*) malloc (sizeof (uint8_t) * 64);
 	bool ok = true;
+	uint8_t* pk = (uint8_t*) malloc (sizeof (uint8_t) * 64);
+	uint8_t* decompressedPoint = (uint8_t*) malloc (sizeof (uint8_t) * 64);
+	uint64_t incorrect = 18446744073709551615U;
```

Comment:
no magic constants please... include `<stdint.h>` and use UINT64_MAX for readability